README.md

```diff
@@ -1,8 +1,161 @@
-[![Build Status](https://travis-ci.org/tgu/kb_uploadmethods.svg?branch=master)](https://travis-ci.org/tgu/kb_uploadmethods)
-
 # kb_uploadmethods
----
 
-Uploaders for kbase datatypes, especially for files in stageing area
+[![Build Status](https://travis-ci.org/tgu/kb_uploadmethods.svg?branch=master)](https://travis-ci.org/tgu/kb_uploadmethods)  
+[Release Notes](RELEASE_NOTES.md)
+
+## Description
+This module implements [KBase](https://www.kbase.us) apps that are used to transform data files into KBase data objects for use in analysis. This also includes a few apps for adding external files to the user's [staging area](https://docs.kbase.us/getting-started/narrative/add-data#uploading-data-from-external-sources), and manipulating them there.
+
+See [here](https://docs.kbase.us/data/upload-download-guide/uploads) for more information on uploading data to KBase.
+
+## Development
```

Comment:
It may be useful to add a comment about how to set up a local development environment. If it is just setting up the kb-sdk local environment, then maybe linking to the local setup instructions for that would be enough.

Reply:
Good idea! Added a link to the SDK install instructions.